src/core/credrank/markovProcessGraph.test.js

```diff
@@ -400,6 +395,16 @@ describe("core/credrank/markovProcessGraph", () => {
       const edgesExpected = edgeOrder.map((x) => mpg.edge(x));
       expect(edgesActual).toEqual(edgesExpected);
     });
+    it("has the correct epoch boundaries for the given intervals", () => {
+      const expected = [-Infinity, 0, 2, Infinity];
+      expect(markovProcessGraph().epochBoundaries()).toEqual(expected);
+    });
+    it("has the right participants", () => {
+      expect(markovProcessGraph().participants()).toEqual([participant]);
+    });
```

Comment:
Is this test related? Maybe meant for a different commit?

Reply:
I was just adding extra tests for consistency (testing all of the "trivial" property accessors)